src/main/java/school/hei/haapi/service/LetterService.java

```diff
@@ -126,7 +126,8 @@ public String getBucketKey(String studentRef, String filename) {
   }
 
   public static String generateRef(String id) {
-    return id.substring(0, 6)
+    return "HEI-"
+        + id.substring(0, 6)
         + "-"
```

Comment:
Please rebase always your branch, in the case that you tell me that your branch is correctly rebased i'll be ok. But here you didn't touch this file right ?

Reply:
actually it was Ilo who asked me to make this modification